### ISSUE TITLE
Heartbeat demo passes on CI despite errors

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#1774]
Original Report:

```
Traceback (most recent call last):
  File "doctor.py", line 13, in <module>
    from nucypher.datastore.keypairs import DecryptingKeypair, SigningKeypair
ModuleNotFoundError: No module named 'nucypher.datastore'
```

> How is it possible the Demo test didn't failed?

> Looking closer here the job appears  to pass but is actually rasing an (unrelated) error, The CircleCI Demos must be misconfigured <https://circleci.com/gh/nucypher/nucypher/75485>

```
TypeError: retrieve() got an unexpected keyword argument 'message_kit'
```

**Comment by KPrasch** [comment]
@vepkenez - <https://circleci.com/gh/nucypher/nucypher/78008>

**Comment by KPrasch** [comment]
Closed in #2581
